Shared/ScriptUpdateFunctions/Invoke-WebRequestWithProxyDetection.ps1

```diff
@@ -47,5 +48,14 @@ function Invoke-WebRequestWithProxyDetection {
         $params = $ParametersObject
     }
 
-    Invoke-WebRequest @params
+    try {
+        Invoke-WebRequest @params
+    } catch {
+        Write-Verbose $_.Exception.Message
+        Write-Verbose "Response: $($_.Exception.Response)"
+        Write-Verbose "Status: $($_.Exception.Status)"
+        Write-Verbose "Stack: $($_.Exception.StackTrace)"
+        Write-Verbose "CategoryInfo: $($_.CategoryInfo)"
+        Write-Verbose "FullyQualifiedErrorId: $($_.FullyQualifiedErrorId)"
```

Comment:
Is there a reason to not use `Shared\Write-ErrorInformation.ps1` here?

Reply:
Will use the shared function in future as I plan to make some general changes to the function to make it shared (#1671).

Reply:
Done.